backends/intel_gpu/CMakeLists.txt

```diff
@@ -0,0 +1,179 @@
+message(FATAL_ERROR "Can't find directory ${ONEAPI_PATH}/${ONEAPI_COMPILER_DIR}")
+endif()
+
+if(NOT DEFINED ENV{CONDA_PREFIX})
+message(FATAL_ERROR "Please install python miniconda ")
+endif()
+
+if(NOT DEFINED CND_INSTALL_DIR)
+    set(CND_INSTALL_DIR "$ENV{CONDA_PREFIX}")
+endif()
+
+if(NOT EXISTS "${CND_INSTALL_DIR}")
+ message(FATAL_ERROR "Can't find conda directory ${CND_INSTALL_DIR}")
+endif()
+
+if(NOT DEFINED ENV{DPL_ROOT})
+	message(FATAL_ERROR "DPL_ROOT ist not set !!! " $ENV{DPL_ROOT})
+endif()
+message(STATUS "Conda DIR : ${CND_INSTALL_DIR}")
+
+# if(NOT DEFINED CND_ENV)
+# set(CND_ENV "$ENV{CONDA_DEFAULT_ENV}")
+# endif()
+
+# message(STATUS "CONDA default env => ${CND_ENV}")
+
+#set(CND_PYTHON "${CND_INSTALL_DIR}/envs/${CND_ENV}/lib/python3.9")
+set(CND_PYTHON "${CND_INSTALL_DIR}/lib/python3.9")
+
+
+if(NOT EXISTS "${CND_PYTHON}")
+   message(FATAL_ERROR "Can't find conda python directory ${CND_PYTHON}, python3.9 required!" )
+endif()
+
+
+
+#############################
+
+set(DPCPP_COMPIER_PATH "${ONEAPI_PATH}/${ONEAPI_COMPILER_DIR}/linux/bin/dpcpp")
+set(ONEAPI_SYCL_INCLUDE "${ONEAPI_PATH}/${ONEAPI_COMPILER_DIR}/linux/include/sycl/")
+set(ONEAPI_SYCL_LIBDIR "${ONEAPI_PATH}/${ONEAPI_COMPILER_DIR}/linux/lib/")
+set(CMAKE_CXX_COMPILER "${DPCPP_COMPIER_PATH}")
+##########################################
+#set(ONEDNN_INC "${PADDLE_BUILD_DIR}/third_party/install/mkldnn/include")
+#set(ONEDNN_LIB "${PADDLE_BUILD_DIR}/third_party/install/mkldnn/lib")
+##########################################
+set(PADDLE_PLUGIN_DIR  "${CND_PYTHON}/site-packages/paddle-plugins/")
+set(PADDLE_INC_DIR     "${CND_PYTHON}/site-packages/paddle/include/")
+set(PADDLE_LIB_DIR     "${CND_PYTHON}/site-packages/paddle/fluid/")
+
+
+#include_directories( ${ONEAPI_SYCL_INCLUDE} ${ONEDNN_INC})
+include_directories( ${ONEAPI_SYCL_INCLUDE} $ENV{DPL_ROOT}/include)
+
+###########################################
+
+
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake")
+
+option(WITH_CUSTOMDNN    "use onednn with custom"     OFF)
+option(WITH_KERNELS    "compile with custom kernels"      ON)
+option(WITH_TESTING    "compile with unit testing"        ON)
+
+set(PLUGIN_NAME        "paddle-custom-intel-gpu")
+set(PLUGIN_VERSION     "0.0.1")
+
+
+
+
+include(paddle)
+message(STATUS "PADDLE_BUILD_DIR DIR : ${PADDLE_BUILD_DIR}")
+message(STATUS "ONEDNN_INC DIR : ${ONEDNN_INC}")
+############ Third-party dependencies
+
+# if(NOT DEFINED PADDLE_BUILD_DIR)
+# set(PADDLE_BUILD_DIR "$ENV{HOME}/projects/Paddle/build")
+# endif()
+
+set(BOOST_INC_DIR      "${PADDLE_BUILD_DIR}/third_party/boost/src/extern_boost")
+set(GFLAGS_INC_DIR     "${PADDLE_BUILD_DIR}/third_party/install/gflags/include")
+set(GLOG_INC_DIR       "${PADDLE_BUILD_DIR}/third_party/install/glog/include")
+set(THREAD_INC_DIR     "${PADDLE_BUILD_DIR}/third_party/threadpool/src/extern_threadpool")
+set(THIRD_PARTY_INC_DIR ${BOOST_INC_DIR} ${GFLAGS_INC_DIR} ${GLOG_INC_DIR} ${THREAD_INC_DIR})
```

Comment:
include(third_party) will add all thrid party headers into include_directories, suppose 117 to 121 is needless.

Reply:
done